exercises/dnd-character/example.js

```diff
@@ -0,0 +1,36 @@
+export class Character {
+  constructor() {
+    this.strength = this.getAbilityScore();
+    this.dexterity = this.getAbilityScore();
+    this.constitution = this.getAbilityScore();
+    this.intelligence = this.getAbilityScore();
+    this.wisdom = this.getAbilityScore();
+    this.charisma = this.getAbilityScore();
+    this.hitpoints = 10 + ability_modifier(this.constitution);
+  }
+
+  getAbilityScore() {
+    let diceRolls = [];
+
+    for (var i = 0; i < 4; i++) {
```

Comment:
The example is _not_ the best solution, so it's fine for this not to be perfect, but just in case someone looks, can you make this `let`